datumaro/cli/commands/compare.py

```diff
@@ -221,44 +219,18 @@ def diff_command(args):
     if args.method is ComparisonMethod.equality:
         if args.ignore_field:
             args.ignore_field = eq_default_if
-        comparator = ExactComparator(
```

Comment:
Why do you remove the existing features?

Reply:
Thank you for comment, I'll revert existed features and combine with new feature.